divio_cli/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '3.3.0'
+__version__ = '3.3.1'
```

Comment:
Same as before, please don't bump the version yet - we'll do that in a separate commit.